README.md

```diff
@@ -59,6 +59,21 @@ This architecture ensures that your content remains exclusively on your device t
 
 6. Click the puzzle piece icon in Chrome toolbar and pin LLMFeeder for easy access
 
+### Option 3: Load in Firefox (Manual)
+
+LLMFeeder can also be loaded as a temporary extension in Firefox for testing and use:
+
+1. Clone this repository:
+   ```
+   git clone git@github.com:jatinkrmalik/LLMFeeder.git
+   ```
+2. Open Firefox and navigate to `about:debugging#/runtime/this-firefox`
+3. Click "Load Temporary Add-on..."
+4. Select the `manifest.json` file inside the `extension` directory.
+5. The extension will appear in your Firefox extensions list for the current session.
+
+> **Note:** Some Chrome APIs may have limited support in Firefox. For best results, use the latest Firefox version.
```

Comment:
TODO: Need to validate full functionality and fix if some feature doesn't work.

Reply:
So there is the main problem of manifest v3, chrome only support it, and firefox support it but do not implemented it fully, so there is a particular version of firefox which support manifest v3, so I have to add it so the user knows that, it is good to run it on latest firefox version.

ref:

https://stackoverflow.com/questions/75043889/manifest-v3-background-scripts-service-worker-on-firefox

https://stackoverflow.com/questions/78491335/how-can-i-write-a-chrome-and-firefox-manifest-v3-extension-that-shares-code-when